templates/bing_coronavirus_query_set/templates.yaml

```diff
@@ -0,0 +1,53 @@
+dataset: bing_coronavirus_query_set
+templates:
+  27e325b3-68d2-4cb4-9418-a683a8870239: !Template
+    id: 27e325b3-68d2-4cb4-9418-a683a8870239
+    jinja: "This query does not talk about the coronavirus explicitly ? \n{{Query}}\n\
+      |||\n{% if IsImplicitIntent %}\nYes\n{% else %}\nNo\n{% endif %}"
```

Comment:
`{% if IsImplicitIntent %}` will always evaluate to True because it's actually a string! You need to do `{% if IsImplicitIntent == "True" %}` I believe.

Reply:
> It's probably worth mentioning in some of the prompts that these are search engine queries (and refer to them as something other than "query" in some cases).

Hey @craffel, thanks for the review. I don't quiet understand what you mean in this comment. Do I need to change the term query ?